docs/edex/new-grid.md

```diff
@@ -15,105 +17,50 @@ If the parameters in the grib file haven't been previously specified, another ch
-        
-    In order to successfully ingest the example file, [**define the appropriate table**](#grib2-products_4).
+### Check HDF5 Data
 
-3. Check that the hdf5 data directory exists for our unnamed grid
+Check that the hdf5 data directory exists for our unnamed grid
 
         ls -latr /awips2/edex/data/hdf5/grid/GribModel:161:0:97
 
-    Though the grib file has been decoded, it has been given a generic name with its center, subcenter, and process IDs (161, 0, 97, respectively). 
+Though the grib file has been decoded, it has been given a generic name with its **center, subcenter, and process IDs** (161, 0, 97, respectively). 
     
 ---
 
 ## Determine Grid Projection
 
-### Grib Products
-
-When the grid was ingested a record was added to the `grid_coverage` table with its navigation information:
-
-    psql metadata
-    
-    metadata=# select nx,ny,dx,dy,majoraxis,minoraxis,la1,lo1,lov,latin1,latin2 from gridcoverage where id=(select distinct(location_id) from grid_info where datasetid='GribModel:7:0:89');
-     nx  | ny  |        dx        |        dy        | majoraxis | minoraxis |       la1        |        lo1        |        lov        |      latin1      |      latin2      
-    -----+-----+------------------+------------------+-----------+-----------+------------------+-------------------+-------------------+------------------+------------------
-     201 | 155 | 4.29699993133545 | 4.29699993133545 |   6378160 |   6356775 | 42.2830009460449 | -72.3610000610352 | -67.0770034790039 | 45.3680000305176 | 45.3680000305176
-    (1 row)
-
-Compare with the projection info returned by wgrib on the original file (look at the bolded sections below and make sure they match up with the corresponding entries returned from the database above):
-<!--
-Using html for this code block so that certain sections within the code can be emphasized (bolded)
--->
-<pre>
-wgrib -V wrf.grib  
-rec 799:27785754:date 2014102318 ALBDO kpds5=84 kpds6=1 kpds7=0 levels=(0,0) grid=255 sfc 6hr fcst: bitmap: 736 undef
-  ALBDO=Albedo [%]
-  timerange 0 P1 6 P2 0 TimeU 1  <b>nx 201 ny 155</b> GDS grid 3 num_in_ave 0 missing 0
-  center 7 subcenter 0 process 89 Table 2 scan: WE:SN winds(grid) 
-  <b>Lambert Conf: Lat1 42.283000 Lon1 -72.361000 Lov -67.077000</b>
-      <b>Latin1 45.368000 Latin2 45.368000</b> LatSP 0.000000 LonSP 0.000000
-      North Pole (201 x 155) <b>Dx 4.297000 Dy 4.297000</b> scan 64 mode 8
-  min/max data 5 21.9  num bits 8  BDS_Ref 50  DecScale 1 BinScale 0
-</pre>
-
-Notice that our grib file has a **Lambert Conformal** projection.  We will need these values for the next step. Note that **there is a tolerance of +/- 0.1 degrees** to keep in mind when defining your coverage area.
-
-### Grib2 Products
-
-When the grid was ingested a record was added to the `grid_coverage` table with its navigation information:
+When a grid is ingested a record is added to the `grid_coverage` table with its navigation information:
```

Comment:
In the psql command, let's add a few more parameters. These are more common for the other projections, but can be helpful. It also prints out the firstGridPointCorner which can be helpful:

here's the new output:

```
metadata=> select nx,ny,dx,dy,majoraxis,minoraxis,la1,lo1,lov,latin1,latin2,spacingunit,lad,la2,latin,lo2,firstgridpointcorner from gridcoverage where id=(select distinct(location_id) from grid_info where datasetid='GribModel:161:0:97');
 nx  | ny  |  dx   |  dy   | majoraxis | minoraxis |    la1    | lo1 | lov | latin1 | latin2 | spacingunit | lad | la2 | latin | lo2 | firstgridpointcorner 
-----+-----+-------+-------+-----------+-----------+-----------+-----+-----+--------+--------+-------------+-----+-----+-------+-----+----------------------
 600 | 640 | 0.005 | 0.005 |           |           | 40.799999 | 261 |     |        |        | degree      |     |     |       |     | UpperLeft
(1 row)
```